bfapi/service/jobs.py

```diff
@@ -208,11 +208,15 @@ def create(
         tide_max_24h=tide_max,
     )
 
-def __genCmd(
-        svc_format: str,
+# The services descended from pzsvc-exec all connect to a CLI (command-line interface) program.
```

Comment:
1. Good to know
2. This function is explicitly for the purpose of creating cmd strings to fit this particular case.  If someone is making a change to this function that causes the comment to become inaccurate, that change itself is a mistake.
2a. "industry wisdom" is in no way unanimous on this subject.